themes/custom/droopler_theme/src/components/d-p-form/d-p-form.twig

```diff
@@ -32,22 +32,33 @@
     </div>
   </div>
 
-  <div {{ bem('content', [], base_class, ['container']) }}>
-    <div {{ bem('content-wrapper', [], base_class, ['row']) }}>
-      <div {{ bem('content-column', ['header'], base_class, [content_classes]) }}>
-        {% block header %}
-        {% endblock %}
+    <div {{ bem('content', [], base_class, ['container']) }}>
+      <div {{ bem('content-wrapper', [], base_class, ['row']) }}>
 
-        {% include '@droopler_theme/base/divider/divider.twig' %}
+        {% set has_header = block('header')|render|striptags('<img>')|trim is not empty %}
+        {% set has_content = block('content')|render|striptags|trim is not empty %}
 
-        {% block content %}
-        {% endblock %}
-      </div>
+        {% if has_header or has_content %}
```

Comment:
Is this condition needed? Is there a possibility of not having both blocks?

Reply:
Yes, there is such possibility. In this case, there's a big empty space next to the form, which looks strange. If this condition is not met, the empty space will not be rendered and the form will be centered.

Reply:
I see, content - header content, not form ;) Get it